lambda.dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM hayd/deno-lambda:1.6.1
+
+ENV DENO_DIR=.deno_dir
```

Comment:
Note: This bug is fixed/isn't required in 1.6.2.

Reply:
good to know!